pkg/network/http/model.go

```diff
@@ -40,6 +40,17 @@ func (k *httpBatchKey) Prepare(n httpNotification) {
 	k.page_num = C.uint(int(n.batch_idx) % HTTPBatchPages)
 }
 
+func (tx *httpTX) postProcess() {
+	b := *(*[HTTPBufferSize]byte)(unsafe.Pointer(&tx.request_fragment))
+	for i := tx.fragmentSize(); i < HTTPBufferSize; i++ {
```

Comment:
may a copy of null bytes (with some boundary check before) would be faster `copy(b[tx.fragmentSize():], zeroHttpBuffer[tx.fragmentSize():HTTPBufferSize])`

`var zeroHttpBuffer [HTTPBufferSize]byte`